Yoast/wordpress-seo
# Remove snapshot tests

## Context


These only cause problems.
The way we want to ensure that it works correctly is via QA and their automated tests.
Currently the snapshots are blocking the CI-pipeline on a more-times-a-day basis which is unacceptable.

## Summary


This PR can be summarized in the following changelog entry:

* [non-user-facing] Removes the snapshot tests from the plugin.

## Relevant technical choices:

* N/A

## Test instructions

This PR can be tested by following these steps:

* run `yarn test` to see all tests pass.

## UI changes

## Documentation

## Quality assurance


Fixes the builds to be passing more.
